lib/plugins/exchange/bitstamp.js

```diff
@@ -0,0 +1,21 @@
+const common = require('../common/ccxt')
+const _ = require('lodash/fp')
+const consts = require('./consts')
+const ORDER_TYPE = consts.ORDER_TYPES.MARKET

```

Comment:
This can be changed into:

```js
const { ORDER_TYPES } = require('./consts')

const ORDER_TYPE = ORDER_TYPES.MARKET
```

This applies for all places where `consts` is being imported, we don't really get a lot of value from having a full `consts` object